### ISSUE TITLE
Now Fishing Animation // Cant edit the files

### ISSUE THREAD

**Issue by TheRealSago** [vk-chase/chase-fishing#2]
Hello, i bought your fishing Script from your Tebex and it works overall, but i cant fix some issues etc. and translate it to my language because it is encrypted. Can u please tell me how to encrypt it so i can full edit it for my Server? Thx

**Comment by TheRealSago** [comment]
Yeah i use all the required scripts but they Emotes not working so with the one rod i have a bucket animation and with the other non animation. And just a couple othe small things that need to be changed to work full with our Server. It would be easier for me if you just can open it, so its normal for scripts that i bought on tebex that u have full access^^ Would be great and for the translation we have our own humor so you cant translate it for us. Hope u understand it

**Comment by vk-chase** [comment]
the props used are base game they are a bucket and toolbox as for the locales i can unescrow those for your editing needs as for the source code that will not be un escrowed.

**Comment by vk-chase** [comment]
give me a day to add all the notifications and such to the locales and check back in your keymaster for a update
